modules/tinymce/CHANGELOG.md

```diff
@@ -66,6 +66,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - The aria labels for the color picker dialog were not translated #TINY-8381
 - The `editor.annotator.remove` did not keep selection when removing the annotation #TINY-8195
 - Dialog labels and other text-based UI properties did not escape HTML markup #TINY-7524
+- The `silver` theme UI was incorrectly rendered before plugins had initialised #TINY-8288
```

Comment:
Not sure it matter should this be `initialized`?

Reply:
Yeah it should be... I always seem to forget the `s` -> `z` for US english 😢

Reply:
Fixed